src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
I understand your point. I thought something like that.

    getContext(context) {
        const _clone = Object.assign(this.constructor);
        const _context = this.contexts[context];

        if (_context.exclude) {
            _context.exclude.forEach((fieldName) => {
                delete _clone.SCHEMA[fieldName];
            })
        }
        return _clone;
    }

about your question `The point is, what user.getContext('astronaut') will return ?`

I think we can return the same type of object but with a new comportament, when I specify that my context should `remove` a specific field, I just return a `new` object without this field , in this approach we keep the same object.